### PR TITLE
Feature/accessibility

### PR DESCRIPTION
**Pull request by Michael-Hanley**
- Adds locate control to tab index
- Adds keyup events
- Renames _onClick to _onButtonAction and updates comment

Issue: #268

### CODE REVIEW

File: src/L.Control.Locate.js [hunk 1]
```diff
@@ -361,49 +365,51 @@ You can find the project at: https://github.com/domoritz/leaflet-locatecontrol
...
-                            var f = this.options.flyTo ? this._map.flyToBounds : this._map.fitBounds;
-                            f.bind(this._map)(this._prevBounds);
-                        }
-                        break;
-                }
-            } else {
-                if (this.options.returnToPrevBounds) {
-                  this._prevBounds = this._map.getBounds();
+        _onButtonAction: function(e) {
+            if (e.type === 'click' || e.key === "Enter" ){
```
Comment (domoritz) [inline src/L.Control.Locate.js]: I have not seen this pattern before. Can you point to some code in Leaflet that does the same for the standard buttons?

File: src/L.Control.Locate.js [hunk 2]
```diff
@@ -280,6 +280,7 @@ You can find the project at: https://github.com/domoritz/leaflet-locatecontrol
             createButtonCallback: function (container, options) {
                 var link = L.DomUtil.create('a', 'leaflet-bar-part leaflet-bar-part-single', container);
                 link.title = options.strings.title;
+                link.tabIndex=0;
```
Comment (domoritz) [inline src/L.Control.Locate.js]: So if we are 0, how do we make sure that other buttons come before in the tab order?

**Review by domoritz — CHANGES_REQUESTED** [review]

**Comment by domoritz** [comment]
I don't see a tab index for the other buttons. Why do they work already?

Screen Shot 2020-05-08 at 17 04 54

File: src/L.Control.Locate.js (hunk 2 above)
Comment (Michael-Hanley) [reply src/L.Control.Locate.js]: tabindex="0" means that the element should be focusable in sequential keyboard navigation, after any positive tabindex values and its order is defined by the document's source order.
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

But it should be a char not a number, I'll make that change.

**Comment by Michael-Hanley** [comment]
Thanks for pointing all that out, turns out this fix was much simpler.

File: src/L.Control.Locate.js (hunk 1 above)
Comment (Michael-Hanley) [reply src/L.Control.Locate.js]: Removed

File: src/L.Control.Locate.js (hunk 2 above)
Comment (Michael-Hanley) [reply src/L.Control.Locate.js]: Removed

**Comment by domoritz** [comment]
Awesome. Thanks for making the change so small.

**Pull request merged**